### PR TITLE
added home component

### PR DESCRIPTION
**Pull request by osegala**

### CODE REVIEW

File: src/assets/images/fav-icon.png
Comment (segalasolutions) [inline src/assets/images/fav-icon.png:1]: This needs to be the favicon of the page.

File: src/app/home-component/home-component.component.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    </div>
+  </footer>
+  
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+  
+  </body>
+  
+  <router-outlet></router-outlet>
```
Comment (segalasolutions) [inline src/app/home-component/home-component.component.html]: Router is what determines which page the subdirectory of the url goes to.

We want the router outlet to be in the app component.

The route localhost: http://localhost:4200/home should navigate to the home page component.

A navigation to  http://localhost:4200 should redirect to  http://localhost:4200/home

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,484 +1 @@
...
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * The content above * * * * * * * * * * * -->
-<!-- * * * * * * * * * * is only a placeholder * * * * * * * * * * -->
-<!-- * * * * * * * * * * and can be replaced. * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-<!-- * * * * * * * * * * End of Placeholder * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-
-<router-outlet></router-outlet>
+<app-home-component></app-home-component>
```
Comment (segalasolutions) [inline src/app/app.component.html]: should be the router outlet here

File: src/app/home-component/home-component.component.html (hunk 1 above)
Comment (segalasolutions) [reply src/app/home-component/home-component.component.html]: Take a look at src/app/app-routing.module.ts. That is where the routes are configured. 

"Angular Router Outlet is a directive that acts as a placeholder in your Angular application where the content of routed components will be displayed. It dynamically renders the component associated with the current route, allowing for a single-page application (SPA) experience by updating the content based on the user's navigation without a full page reload."

Decent website explination: https://v17.angular.io/tutorial/tour-of-heroes/toh-pt5

There are probably good youtube videos too.

Comment (osegala) [reply src/app/home-component/home-component.component.html]: i believe ive fixed it now

**Pull request merged**